adversarial_apps/src/components/RecentOwnership.tsx

```diff
@@ -90,7 +102,20 @@ export const RecentOwnership = (props: Props) => {
     return (
         <div >
             <br />
-            <h2 className="text-2xl font-semibold text-white mb-4">Recent Owners:</h2>
+            <h2 className="text-2xl font-semibold text-white mb-4">Ownership Tracking:</h2>
+            <br />
+            <p>
+                Below is a table of most recent changes in ownership of the company as provided by Form 4 submissions if there are any.
+                This provides a live updated view of how ownership is changing in real time, as our risk score is only calculated on data given once per year from the annual DEF 14A form.
+                <br></br>
+                <br></br>
+                The letter relates to the exact type of transaction that has occurred. For covnince we summed up the transaction codes and provided the key:
+            </p>
+            <ul className="list-disc ml-5">
+                <li>If the transaction code letter has a + that means the Receiver has obtained more control over the company.</li>
+                <li>If the transaction code letter has a - that means the Receiver has lost or given up control over the company.</li>
+                <li>If the transaction code letter has a ~ that means the Receiver has not changed their ownership of the company.</li>
```

Comment:
nitpick: would like if the +,-,~ were bolded or in quotes like this to help them stand out:
```suggestion
                <li>If the transaction code letter has a "+" that means the Receiver has obtained more control over the company.</li>
                <li>If the transaction code letter has a "-" that means the Receiver has lost or given up control over the company.</li>
                <li>If the transaction code letter has a "~" that means the Receiver has not changed their ownership of the company.</li>
```